stdlib/public/Distributed/DistributedActorSystem.swift

```diff
@@ -170,74 +281,9 @@ func _executeDistributedTarget(
   on actor: AnyObject, // DistributedActor
   _ targetName: UnsafePointer<UInt8>, _ targetNameLength: UInt,
   argumentBuffer: Builtin.RawPointer, // HeterogeneousBuffer of arguments
-  resultBuffer: Builtin.RawPointer
+  resultBuffer: Builtin.RawPointer?
```

Comment:
@ktoso I don't think we can do this, since `Optional` type has different ABI (on 64-bit it's a `struct { i64, i64 }` for example). I think you either should make it just a `RawPointer` or I need to change signature of accessor to accept exploded struct for a result buffer...

Reply:
Actually maybe this is a false alarm... I use multiple places where UnsafeRawPointer is used as an optional in the stdlib

Reply:
Might be worth it to just switch over to `UnsafeRawPointer?` for both `argumentBuffer:` and `resultBuffer:` to keep it consistent with other APIs e.g. ones from Misc.swift

Reply:
Not sure which way to go -- we use Builtin.RawPointer a lot in all of the concurrency APIs so it's consitent with those I guess... but this is the metadata ones I suppose... Let's revisit this once we confirm we got it right -- still trying to get it not crash the invocation... :)